ms365_accountcreator/translations/de/LC_MESSAGES/messages.po

```diff
@@ -101,13 +111,18 @@ msgid ""
 "I consent that my st****** email is saved to prevent duplicate Account "
 "creation."
 msgstr ""
-"Ich stimme zu, dass meine st****** E-Mail Adresse gespeichert wird um zu "
-"verhindern, das mehrere Accounts für einen Student angelegt werden "
-"können."
+"Ich stimme zu, dass meine eingegebene st****** E-Mail Adresse verarbeitet"
+" und gespeichert wird um das generierte Passwort zu verschicken und um "
+"sicherzustellen, dass maximal ein Account pro Studi angelegt wird. Mir "
+"ist bewusst, dass ich diese Zustimmung widerrufen und die Löschung der "
+"Daten veranlassen kann."
```

Comment:
Die detaillierte beschreibung sollte vielleicht nicht komplett in den label. Deshalb hab ich ja die "Detailed legal text *" drin.

Reply:
Hm ja aber dann gehört es nicht mehr zu dem Haken und damit "unterschreibt" man es nicht

Reply:
Dann vielleicht den zweck auslassen und in nur der description dazu näher erläutern. Die description ist auch noch mit dem Form Feld verlinkt (auch für screenreader). Eventuell kann man die description auch in den label packen. Dann hat man den unterschied fett und nicht fett und alles ist klickbar.

Reply:
Ich find das jetzt echt nicht so schlimm, dass der Fette Text etwas länger ist, Die Website muss kein Design Contest gewinnen.
So stimmt man demVerwendungszweck und der Verwendungsart (+Ort) zu und "unterschreibt" auch das mit dem Widerrufen.
Und zusätzliche infos über die verarbeitung stehen dann drunter.

Reply:
ok